### DISCUSSION TITLE
Nested span loosing CSS property

### DISCUSSION THREAD

**Discussion by ghlLarney** [ueberdosis/tiptap#5213] [General Questions]
When the HTML is injected into TipTap using the setHtml function, the structure is altered, causing text2 to lose its color and font properties. This issue is critical because we are transitioning from TinyMCE to TipTap, and a significant amount of legacy HTML is already stored in our database. This transition must be seamless to preserve the integrity of our existing content. Immediate guidance and a solution are essential to ensure that we maintain the correct formatting and appearance of our content during this migration. Please advise on how we can address this problem effectively.

**Raw**
```html
<p>
    <span style="font-size: 14pt; font-family: arial, helvetica, sans-serif;">
        text1 
        <span style="color: #3366ff;">
            text2
        </span> 
        text3
        <br />
        <br />
        text 4
    </span>
</p>
```

**In tiptap**

```html
<p>
    <span style="font-size: 14pt; font-family: arial, helvetica, sans-serif;">
        text1 
    </span>
    <span style="color: #3366ff;">
        text2
    </span> 
    text3
    <br />
    <br />
    text4
</p>
```

Previous discussion that weren't helpful
https://github.com/ueberdosis/tiptap/discussions/1919
https://github.com/ueberdosis/tiptap/discussions/1815

**Comment by ghlLarney** [comment]
It looks like ProseMirror is stripping everything after first child element of a span. If that how it works? What could be a workaround using tiptap extension? I've tried Mark with inline: true as well. Nothing works.

**Comment by bdbch** [comment]
Let's continue the discussion of this in #1815
